spec/helpers/pipeline_runs_helper_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+require "webmock/rspec"
+
+RSpec.describe PipelineRunsHelper, type: :helper do
+  describe "#new_failed_pipeline_run" do
+    def create_new_failed_pipeline_run(sample)
+      pr = helper.new_failed_pipeline_run(sample)
+      pr.error_message = "Failed Pipeline Run"
+      pr.save
+      return pr
+    end
+
+    before do
+      # Create a dummy alignment config for the pipeline run.
+      alignment_config = create(:alignment_config)
```

Comment:
I haven't done this but I think Tiago would recommend using spec/factories/alignment_configs factories?